### ISSUE TITLE
Task with path 'installApp' not found in project

### ISSUE THREAD

**Issue by betsalel-williamson** [mini2Dx/parcl#5]
Was attempting to add this to my project and got the error: 

```
* What went wrong:
Could not determine the dependencies of task ':project:bundleNative'.
> Task with path 'installApp' not found in project ':project'.
```

Followed directions to add the `buildscript` and `project(":project") {` as listed. Have the project included in the root `settings.gradle` file.

**Comment by tomcashman** [comment]
Which verson of parcl are you using? This could be the same issue as #2

**Comment by tomcashman** [comment]
Fixed in 1.0.12. Current version is 1.2.1.
